webportal/webapp/src/app/account/loginsync/guards/loginsync.guard.spec.ts

```diff
@@ -1,31 +1,100 @@
 import { LoginSyncGuard } from './loginsync.guard';
+import { CognitoService } from '../../../../services/cognito.service';
```

Comment:
Boy scout rule... my changes were minor to the actual file, but I added tests for it.